src/level_set/RelaxationLSMethod.cpp

```diff
@@ -663,7 +786,122 @@ RelaxationLSMethod::relax(Pointer<CellData<NDIM, double> > dist_data,
     }
 
     return;
-} // relax
+} // relaxGS
+
+void
+RelaxationLSMethod::relaxRK(Pointer<CellData<NDIM, double> > dist_data,
+                            const Pointer<CellData<NDIM, double> > dist_init_data,
+                            const Pointer<Patch<NDIM> > patch,
+                            const int stage) const
```

Comment:
What's going on with `stage` here? This value isn't read. Could you add some documentation explaining what it might be intended to do in the future?

Reply:
I'll add that this could be used in the future to do RK3, etc.